python/ray/llm/tests/batch/cpu/processor/test_processor_base.py

```diff
@@ -188,6 +188,110 @@ def overrider(name: str, stage: StatefulStage):
     )
 
 
+def test_builder_kwargs_passthrough():
```

Comment:
Could you spearate out these test scenarios into different test methods? Maybe create a Test class to capture them all and share resource between them using fixtures, etc.